app/views/hyrax/dashboard/collections/_form_discovery.html.erb

```diff
@@ -8,8 +8,8 @@
   <p><%= t('.para2') %></p>
 
   <%# OVERRIDE: add conditional title %>
-  <div class="form-group" <% if cannot?(:manage_discovery, @collection) %>title="<%= t('permissions.collections.cannot.manage_discovery') %>"<% end %>>
-    <label class="radio">
+  <div class="form-check" <% if cannot?(:manage_discovery, @collection) %>title="<%= t('permissions.collections.cannot.manage_discovery') %>"<% end %>>
```

Comment:
Curious how form-check instead of radio even though we have a radio button.

Reply:
Good point, going to modify